tools/spec-gen-sdk/src/utils/runScript.ts

```diff
@@ -146,7 +146,7 @@ const listenOnStream = (
       }
     }
     setSdkAutoStatus(result, lineResult);
-    if (context.config.runEnv === 'azureDevOps' && line.toLowerCase().includes("error")) {
+    if (context.config.runEnv === 'azureDevOps' && line.toLowerCase().includes("[error]")) {
```

Comment:
This looks strange

Reply:
This is the prefix format reported by each respective language script. Here is an [example run](https://dev.azure.com/azure-sdk/playground/_build/results?buildId=4555392&view=logs&j=83516c17-6666-5250-abde-63983ce72a49&t=c10d5f44-55ce-55d7-7975-407ed75d9a96&l=191)